src/doomstat.h

```diff
@@ -245,6 +245,11 @@ typedef struct
 	UINT8 numGradedMares;   ///< Internal. For grade support.
 	nightsgrades_t *grades; ///< NiGHTS grades. Allocated dynamically for space reasons. Be careful.
 
+	// miru: to use custom mapheaderinfo options, we need to add them to the struct first
+	UINT8 levelwipe;
+	UINT8 postlevelwipe;
+	UINT8 wipecolor;
```

Comment:
this is no fault of your own but level headers probably need to start being handled in a more resource efficient way in general, and this is an example since many maps will not use these babies at all